## [float] leaf size of kd-tree in clustering algorythm units -> meters
leaf_size: 0.01
##
FOV_side_half_dist: 0.15
##
FOV_min_dist: 0.4
##
FOV_max_dist: 3.0
##
Street_level: -0.02
##
cluster_tolerance: 0.05
##
min_cluster_size: 100
##
max_cluster_size: 25000